.test-infra/metrics/docker-compose.yml

```diff
@@ -44,11 +62,16 @@ services:
       - GF_SECURITY_ADMIN_PASSWORD=<GrafanaPasswordHere>
       - GF_AUTH_ANONYMOUS_ENABLED=true
       - GF_AUTH_ANONYMOUS_ORG_NAME=Beam
-      - DB_HOST=beampostgresql
-      - DB_PORT=5432
-      - DB_DBNAME=beam_metrics
-      - DB_DBUSERNAME=admin
-      - DB_DBPWD=<PGPasswordHere>
+      - PSQL_DB_HOST=beampostgresql
```

Comment:
Minor: should we use "user_metrics_db" and "perftests_metrics_db" instead?

Reply:
Do you mean changing databases names (PSQL_DB_DBNAME and INFLUX_DB_DBNAME) or hostnames (PSQL_DB_HOST and INFLUX_DB_HOST)?